apps/sms/js/thread_ui.js

```diff
@@ -1935,20 +1933,17 @@ var ThreadUI = {
     }
   },
 
-  checkInputs: function thui_checkInputs() {
+  updateSelectionStatus: function thui_updateSelectionStatus() {
```

Comment:
Although entering edit mode while the thread is still loading is a rare case, I agree it's still nice if we could refine this part.